src/optimize/base_optimizer.js

```diff
@@ -269,7 +268,7 @@ export default class BaseOptimizer {
         filename: '[name].bundle.js',
         sourceMapFilename: '[file].map',
         publicPath: PUBLIC_PATH_PLACEHOLDER,
-        devtoolModuleFilenameTemplate: '[absolute-resource-path]',
+        devtoolModuleFilenameTemplate: info => `webpack:///${ path.relative(fromRoot('.'), info.absoluteResourcePath) }`,
```

Comment:
What's the benefit of the `webpack:///` prefix? I feel like devtools is cleaner without it...

Reply:
@spalger it was only to try to keep the same behaviour we had on the devtools before. With `'[absolute-resource-path]'` setting the sourcemapped files will be listed under an absolute folder to my local computer kibana installation, with `webpack:///` prefix they will be listed under a folder with the same name, without the prefix they will be listed on the devtools under the `localhost:port` folder along with the other files. Do u fill it more organised without the prefix? I'm okay with both 😄

Reply:
Yeah, I'd personally prefer to avoid the `webpack:` prefix bit, I'm all good with switching to relative paths.

Reply:
👍